api/idea-card/models/idea-card.settings.json

```diff
@@ -0,0 +1,59 @@
+{
+  "kind": "collectionType",
+  "collectionName": "idea_cards",
+  "info": {
+    "name": "IdeaCard"
+  },
+  "options": {
+    "increments": true,
+    "timestamps": true,
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "ideaName": {
+      "type": "string",
+      "required": true,
+      "unique": true
+    },
+    "tagline": {
+      "type": "text",
+      "required": true
+    },
+    "description": {
+      "type": "text",
+      "required": true
+    },
+    "targetAudience": {
+      "type": "text",
+      "required": true
+    },
+    "timeCommitmentMin": {
+      "type": "integer",
+      "required": true
+    },
+    "timeCommitmentMax": {
+      "type": "integer",
+      "required": true
+    },
+    "difficultyLevel": {
+      "type": "string",
+      "required": true
+    },
+    "skills": {
+      "type": "component",
+      "repeatable": true,
+      "component": "skills.skills"
+    },
+    "openPositions": {
+      "type": "component",
+      "repeatable": true,
+      "component": "positions.open-positions"
+    },
+    "user": {
```

Comment:
What user is this?

Reply:
this is the user from the users-permissions plugin. This sets up the one to many relationship between the user and idea cards

Reply:
What does user mean here? Users interested in the idea?

Reply:
Users are able to create idea cards. This just means that a user can create multiple idea cards

Reply:
I see, can we make it clear that this field is the user that created the idea?

Reply:
How should I make that clear?

Reply:
Maybe call this field author?

Reply:
I see. What's the problem with it just being user? I figured this implied the relationship between the user and the idea cards